pkg/codegen/testing/test/testdata/simple-methods-schema/schema.json

```diff
@@ -53,13 +53,13 @@
             "plain": true
           },
           "name": {
-            "$ref": "/random/v2.3.1/schema.json#/resources/random:index%2FrandomPet:RandomPet"
+            "$ref": "/random/v4.3.1/schema.json#/resources/random:index%2FrandomPet:RandomPet"
```

Comment:
Why'd this need to change? (Just curious)

Reply:
Apparently this older version of the random provider didn't include arm64 builds, so it was failing on my M1 macbook.